marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -471,8 +471,8 @@ describe('RTL Source-to-entity map tests', () => {
         //Verify Test button click
         fireEvent.click(getByText('Test'))
         await(waitForElement(() => getByTestId('propName-value')))
-        expect(getByTestId('propName-value')).toHaveTextContent('user1@marklogic.com,user2... (300 more)')
-        expect(getByTestId('propAttribute-value')).toHaveTextContent('u@ml.com,v@ml.com... (30 more)')
+        expect(getByTestId('propName-value')).toHaveTextContent('extremelylongusername@m...')
+        expect(getByTestId('propAttribute-value')).toHaveTextContent('s@ml.com t@ml.com (6 more)')
```

Comment:
Previously multiple values showed up in one line so the assertion then made sense. Does the test now just need a space to match new line text?

Reply:
Can we also add an assertion on mouse over (x more)?

Reply:
@bsrikan Yes, when I ran the tests, it seemed that the white space matches new line text content in the UI, so the test passes with my implementation.

Reply:
@bsrikan For mouse over on (x more), do you intend to test the tooltip that shows up? Currently there is no spec for any mouse interaction with (x more) specifically. But mouseover on the entire cell value will bring up a tooltip so I can add an assertion for that?

Reply:
Right @briantang a test for the tooltip that shows all values on mouse over.